docs/operating/monitoring.md

```diff
@@ -6,13 +6,17 @@ sidebar_position: 2
 You can monitor your Quickwit cluster with Grafana.
 
 We provide two Grafana dashboards to help you monitor:
-- [indexers performance](https://github.com/quickwit-oss/quickwit/blob/main/monitoring/grafana/dashboards/indexer.json)
+- [indexers performance](https://github.com/quickwit-oss/quickwit/blob/main/monitoring/grafana/dashboards/indexers.json)
+- [indexers performance](https://github.com/quickwit-oss/quickwit/blob/main/monitoring/grafana/dashboards/searchers.json)
 - [metastore queries](https://github.com/quickwit-oss/quickwit/blob/main/monitoring/grafana/dashboards/metastore.json)
 
 Both dashboards relies on a prometheus datasource fed with [Quickwit metrics](../reference/metrics.md).
 
 ## Screenshots
 
-![Indexer Grafana Dashboard](../assets/images/screenshot-indexer-grafana-dashboard.jpeg)
+![Indexers Grafana Dashboard](../assets/images/screenshot-indexers-grafana-dashboard.jpeg)
+
+
+![Searchers Grafana Dashboard](../assets/images/screenshot-searchers-grafana-dashboard.jpeg)
 
 ![Metastore Grafana Dashboard](../assets/images/screenshot-metastore-grafana-dashboard.png)
```

Comment:
Technically, with Pg, you could have multiple metastores :) Same for Janitor, we need to make them HA eventually.